pkg/clusteragent/admission/mutate/common/common.go

```diff
@@ -175,3 +175,17 @@ func ShouldMutatePod(pod *corev1.Pod) bool {
 
 	return config.Datadog.GetBool("admission_controller.mutate_unlabelled")
 }
+
+// ContainerRegistry gets the container registry config using the specified
+// config option, and falls back to the default container registry if no webhook-
+// specific container registry is set.
+func ContainerRegistry(specificConfigOpt string) string {
+	var containerRegistry string
```

Comment:
Nit: this can be simplified a bit, we don't really need this var.
```go
	if config.Datadog.IsSet(specificConfigOpt) {
		return config.Datadog.GetString(specificConfigOpt)
	}

	return config.Datadog.GetString("admission_controller.container_registry")
```